packages/react/src/components/Modal/Modal.base.tsx

```diff
@@ -228,6 +229,18 @@ export const ModalBase: React.FunctionComponent<IModalProps> = React.forwardRef<
       [keepInBounds, internalState],
     );
 
+    /**
+     * Determines the modal role for accessibility
+     * The isModeless and isBlocking will be ignored only when isAlert is set
+     */
+    const getDialogRole = () => {
+      if (isAlert || isModeless === false || isBlocking) {
```

Comment:
I think this should be:

```suggestion
      if (isAlert || (!isModeless && isBlocking)) {
```

Reply:
Agree, thanks for catching this